### DISCUSSION TITLE
Flameshot version in Arch is out of date

### DISCUSSION THREAD

**Discussion by 07Excel** [flameshot-org/flameshot#2299] [Q&A]
When does the version 11 come in the Arch repos?

**Comment by mmahmoudian** [comment]
We do not maintain the community repo of Arch, but we do maintain the AUR (`flameshot-git`) which is marked by the pink box and arrow:.

![image](https://user-images.githubusercontent.com/390889/150525469-ee9f2411-57cd-4c87-8cc8-0ed707299c38.png)

To get info about the maintainer of the community repo, you can use 

```sh
pacman --sync --info flameshot
```
which at the time of writing this will return:

>```
> Repository      : community
> Name            : flameshot
> Version         : 0.10.2-1
> Description     : Powerful yet simple to use screenshot software
> Architecture    : x86_64
> URL             : https://github.com/flameshot-org/flameshot
> Licenses        : GPL
> Groups          : None
> Provides        : None
> Depends On      : qt5-svg  hicolor-icon-theme
> Optional Deps   : gnome-shell-extension-appindicator: for system tray icon if you are using Gnome
> Conflicts With  : None
> Replaces        : None
> Download Size   : 558.42 KiB
> Installed Size  : 2069.42 KiB
> Packager        : Jiachen YANG <farseerfc@archlinux.org>
> Build Date      : Fri 26 Nov 2021 05:06:37 PM EET
> Validated By    : MD5 Sum  SHA-256 Sum  Signature
>```
